Curs/src/style/login.less

```diff
@@ -221,34 +352,105 @@ a {
 @media screen and (max-width: 1200px) and (min-width: 992px) {
     /* Styles */
 
-    .footer-section {
-        text-align: center;
+    #alert {
+        color: red;
+        font-size: 20px;
+        width: 97%;
     }
 
     .feedback-image {
-        float: left !important;
+        width: 43%;
+        margin-left: 325px;
+        height: 300px;
+    }
+
+    .right-text-description {
+        width: 100%;
+        margin-top: 350px;
+        text-align: center;
+        float: none;
+    }
+
+    .left-text-description {
+        width: 100%;
+        margin-top: 100px;
+        text-align: center;
+        float: left;
+        padding-left: 0;
     }
 
+    .text-description-2 {
+        margin-top: 0;
+    }
+
+    .right-image {
+        margin-bottom: 50px;
+        margin-top: 200px;
+        width: 100%;
+        margin-left: 0;
+    }
+
+    .footer-section {
+        text-align: center;
+    }
+
+
     .header-form-section {
         width: 34%;
         margin: 0 auto;
     }
 
-    .right-image {
-        float: left;
-    }
 }
 
 /*Screen between 1200px and 1824px*/
 @media screen and (max-width: 1824px) and (min-width: 1200px) {
     /* Styles */
 
-    .header-form-section {
-        width: 24%;
-        margin: 0 auto;
+    #alert {
```

Comment:
Alright, OK, I get it... but didn't we declare these styles a couple times above?

Reply:
Yes I did. But I declare it for different break-points. Exists a better practice?

Reply:
What you _could_ do is declare the common styling in a more general responsive block, e.g. if you wanted this styling to apply for screen sizes up to 1024px wide, you'd just write it once in a `@media max-width: 1024px` block and you're done - no need to repeat yourself for `min-width: 480px and max-width: 640px`, `min-width: 640px and max-width: 860px` and so on.